articles/app-service/app-service-web-tutorial-custom-ssl.md

```diff
@@ -308,6 +308,8 @@ New-AzureRmWebAppSSLBinding `
     -CertificatePassword <PFX_password> `
     -SslState SniEnabled
 ```
+## Public Certificates (Optional)
+You can upload [Public Certificates](https://blogs.msdn.microsoft.com/appserviceteam/2017/11/01/app-service-certificates-now-supports-public-certificates-cer/) to your web app . You can use Public Certifictes with Web Apps on App Service Environment (ASE) or non-ASE web apps. If you require to store the certificate in LocalMachine certificate Store , you need use a web app on App Service Enviroment. For more details , See [How to configure Public Certificate to your Web App](https://blogs.msdn.microsoft.com/appserviceteam/2017/11/01/app-service-certificates-now-supports-public-certificates-cer).
```

Comment:
- [ ] There is a space before a period on the first sentence, can you remove the space
- [ ] non-ASE web app is not official guidance, can you update to say App Service or App Service Environment
- [ ] You've checked in an image, did you want to reference that image somewhere in this section?

Reply:
I made the changes you listed above. Removed unnecessary images from media folder . I included one image for public cert upload UI